src/Symfony/Component/Console/Command/Command.php

```diff
@@ -678,6 +681,11 @@ public function asXml($asDom = false)
         return $output->fetch();
     }
 
+    public function setInputBound($inputBound)
```

Comment:
I'm not really happy about that but adding this method seems the only way to fix it without breaking anything (changing the way the input is bound would do, never binding the input from the command too).

> I don't get what this does without reading twice or more the code itself

That's the console... application change the state of commands all the time, ugly but actually necessary. https://github.com/symfony/symfony/issues/19441#issuecomment-264635448 explain well what happens, looking at it may make this easier to read